ouroboros-consensus/src/unstable-consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Adversarial.hs

```diff
@@ -637,15 +637,29 @@ withinYS (Delta d) !mbYS !(RI.Race (C.SomeWindow Proxy win)) = case mbYS of
 --
 -- The count will be strictly smaller than the number of active slots in the given 'ChainSchema'.
 --
+-- The following precondition ensures that there are enough slots to produce an
+-- alternative chain schema with at least k+1 slots.
```

Comment:
Did you mean "active slots"?

```suggestion
-- alternative chain schema with at least k+1 active slots.
```

Reply:
Why do we want to ensure the alternative chain has at least k+1 (active) slots?

Reply:
(see https://github.com/input-output-hk/ouroboros-consensus/pull/477#discussion_r1401765035)